lib/rules/lint-eol-last.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+const Rule = require('./base');
+
+module.exports = class EolLast extends Rule {
```

Comment:
This should implement a `parseConfig` so that we can enforce valid config before beginning linting any templates.

Reply:
done with tests